packages/venia-concept/package.json

```diff
@@ -88,6 +89,9 @@
     "graphql-tag": "~2.10.1",
     "html-webpack-plugin": "~3.2.0",
     "informed": "~2.1.13",
+    "jarallax": "~1.11.1",
+    "load-google-maps-api": "~2.0.1",
+    "lodash.escape": "~4.0.1",
```

Comment:
I'm inclined to declare as many dependencies as possible as `peerDependencies`, which is what they are in `@magento/pagebuilder`. That's a safer model, to me, for building an efficient app. It enforces that only one copy of `jarallax` and other such libraries can be in the same bundle at one time.

When a dev installs `@magento/pagebuilder`, they'll see warnings that they also need to add these packages as dependencies. So then they can, and they'll have control over those package versions. If another package wants an incompatible version of `jarallax`, it'll fail the build, rather than silently putting two copies of the same library into the bundle.

Reply:
Ah, I see why. If these are both peer dependenies *and* dev dependencies in `@magento/pagebuilder`, that does explain why they would be listed as dependencies of `@magento/venia-concept`. Wouldn't they be real dependencies here, though, rather than dev dependencies?

Reply:
@zetlen if the application did not have `@magento/pagebuilder` installed, would the bundle still include Jarallax etc? Or would these only be included if the bundler discovered their usage in the application?

Reply:
@davemacaulay That is an interesting question. In this peerDeps driven workflow, here's what the workflow would be:

1. Without PageBuilder installed, none of those other deps are installed either
1. `yarn add @magento/pagebuilder`
1. Install process shows warnings about installing `jarallax` and other dependencies
1. Build fails if those dependencies are not installed
1. Developer must install those deps as well

This happens with Babel and other common tools, but I would understand if it feels too complex for a new NPM/yarn user. There may be other good ways to prevent multiple copies of a library in the same bundle; do you have an alternative suggestion?

Reply:
@zetlen I guess I'm used to the world of composer where it'll automatically resolve and install all dependencies on install, but you're right in saying if they required different versions things would get a little messy.

I don't have an alternative and if this is the way to go it makes enough sense to me. I'd be inclined to say we shouldn't include the deps for Page Builder in `venia-concept`, as any other package not authored by us won't have this luxury, but as we're going to include `@magento/pagebuilder` by default for now we probably have too.

If you have any suggestions of how we can avoid this that'd be great, but not necessary for this iteration.